CHANGELOG_PENDING.md

```diff
@@ -5,4 +5,6 @@
 - Fixes failing pulumi refresh on org tokens with a slash in its name [#388](https://github.com/pulumi/pulumi-pulumiservice/issues/388)
 - Fix panic when using computed values in environment definition [#411](https://github.com/pulumi/pulumi-pulumiservice/issues/411)
 
+- Fix failing build due to conflicting resource names [#408](https://github.com/pulumi/pulumi-pulumiservice/issues/408)
```

Comment:
Not sure on conventions here but I feel like generally changelogs don't include things like example or test updates